bower.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Thanks!  I added the license and version (0.1.0) in addition to an actual license and authors files.